docs.Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine
+
+RUN apk --update upgrade \
+&& apk --no-cache --no-progress add py-pip \
+&& rm -rf /var/cache/apk/*
+
+RUN mkdir /mkdocs
+
+WORKDIR /mkdocs
+COPY requirements.txt /mkdocs
+
```

Comment:
You don't have to create mkdocs directory with the instruction `RUN mkdir /mkdocs`, the `COPY` instruction create the directory automatically.